src/DssSpell.t.sol

```diff
@@ -361,41 +361,41 @@ contract DssSpellTest is DssSpellTestBase {
         assertTrue(lerp.done());
     }
 
-    function testNewChainlogValues() public { // don't disable
+    function testNewChainlogValues() private { // amke private to disable
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // RWA015
-        _checkChainlogKey("RWA015_A_JAR");
-        _checkChainlogKey("RWA015");
-        _checkChainlogKey("MCD_JOIN_RWA015_A");
-        _checkChainlogKey("RWA015_A_URN");
-        _checkChainlogKey("RWA015_A_INPUT_CONDUIT_URN");
-        _checkChainlogKey("RWA015_A_INPUT_CONDUIT_JAR");
-        _checkChainlogKey("RWA015_A_OUTPUT_CONDUIT");
-        _checkChainlogKey("PIP_RWA015");
+        // _checkChainlogKey("RWA015_A_JAR");
+        // _checkChainlogKey("RWA015");
+        // _checkChainlogKey("MCD_JOIN_RWA015_A");
+        // _checkChainlogKey("RWA015_A_URN");
+        // _checkChainlogKey("RWA015_A_INPUT_CONDUIT_URN");
+        // _checkChainlogKey("RWA015_A_INPUT_CONDUIT_JAR");
+        // _checkChainlogKey("RWA015_A_OUTPUT_CONDUIT");
+        // _checkChainlogKey("PIP_RWA015");
 
         _checkChainlogVersion("1.14.13");
     }
 
-    function testNewIlkRegistryValues() public { // make private to disable
+    function testNewIlkRegistryValues() private { // make private to disable
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // Insert new ilk registry values tests here
         // RWA015
-        (, address pipRwa015,,) = oracle.ilks("RWA015-A");
-
-        assertEq(reg.pos("RWA015-A"),    63);
-        assertEq(reg.join("RWA015-A"),   addr.addr("MCD_JOIN_RWA015_A"));
-        assertEq(reg.gem("RWA015-A"),    addr.addr("RWA015"));
-        assertEq(reg.dec("RWA015-A"),    GemAbstract(addr.addr("RWA015")).decimals());
-        assertEq(reg.class("RWA015-A"),  3);
-        assertEq(reg.pip("RWA015-A"),    pipRwa015);
-        assertEq(reg.name("RWA015-A"),   "RWA015-A: BlockTower Andromeda");
-        assertEq(reg.symbol("RWA015-A"), GemAbstract(addr.addr("RWA015")).symbol());
+        // (, address pipRwa015,,) = oracle.ilks("RWA015-A");
```

Comment:
Just private the function, no need to comment these out

Reply:
Given this context, I referred to the checklist and [this](https://github.com/makerdao/pe-checklists/blob/master/spell/spell-crafter-mainnet-workflow.md?plain=1#L25) appears to be a test covered by the policy of removing setup (explicitly mentioned as an example).

Given that the RWA015 content is coming from the previous PR (this week's spell).

`testCollateralIntegration` is also included as an example here, however this was not used in the 2023-06-14 spell, so I am OK if you want to keep the content in there (commented out) or replace it with one of the two options:

1) Empty (just the line `// Insert new collateral tests here` and nothing after it)

2) The following template code
```
// Insert new collateral tests here
        checkIlkIntegration(
             "TOKEN-X",
             GemJoinAbstract(addr.addr("MCD_JOIN_TOKEN_X")),
             ClipAbstract(addr.addr("MCD_CLIP_TOKEN_X")),
             addr.addr("PIP_TOKEN"),
             true,
             true,
             false
        );
```

To my knowledge, this will not break the tests as we are currently encountering.

Reply:
We have several examples of collaterals that were integrated at some point and left there aftwerwad:

See:
- [2023-03-11](https://github.com/makerdao/spells-mainnet/blob/a3d6cc06ef987689a753e4f1707c20a0b61c82c3/archive/2023-03-11-DssSpell/DssSpell.t.sol#L208-L223)
- [2023-04-26](https://github.com/makerdao/spells-mainnet/blob/a3d6cc06ef987689a753e4f1707c20a0b61c82c3/archive/2023-04-26-DssSpell/DssSpell.t.sol#L186-L201)
- [2022-10-26](https://github.com/makerdao/spells-mainnet/blob/a3d6cc06ef987689a753e4f1707c20a0b61c82c3/archive/2022-10-26-DssSpell/DssSpell.t.sol#L95-L109)
- [2023-02-03](https://github.com/makerdao/spells-mainnet/blob/a3d6cc06ef987689a753e4f1707c20a0b61c82c3/archive/2023-02-03-DssSpell/DssSpell.t.sol#L180-L195)
- [2023-04-05](https://github.com/makerdao/spells-mainnet/blob/a3d6cc06ef987689a753e4f1707c20a0b61c82c3/archive/2023-04-05-DssSpell/DssSpell.t.sol#L186-L201)
- [2023-02-08](https://github.com/makerdao/spells-mainnet/blob/a3d6cc06ef987689a753e4f1707c20a0b61c82c3/archive/2023-02-08-DssSpell/DssSpell.t.sol#L180-L195)

So this is as legit as your suggestions if we are using the precedents set by the archive.

Reply:
Yeah `testCollateralIntegration` still has `GNO-A` in it, which is extraneous to the cleanup from 2023-06-14, so that makes sense. The `testNewIlkRegistryValues` from the RWA015 example looks good to me - resolving this issue.